sky/jobs/utils.py

```diff
@@ -124,6 +129,102 @@ def terminate_cluster(cluster_name: str, max_retry: int = 6) -> None:
             time.sleep(backoff.current_backoff())
 
 
+# Use LRU Cache so that the check is only done once.
+@annotations.lru_cache(scope='request', maxsize=1)
+def is_consolidation_mode() -> bool:
```

Comment:
Can we add a comment on this function explaining what the consolidation mode is and how it works?